### ISSUE TITLE
Show Song Cover feature not Functioning 

### ISSUE THREAD

**Issue by Suchait05** [vfsfitvnm/ViMusic#1281]
### Steps to reproduce the bug

1. Go to 'ViMusic on Android'
2. Click on 'triple vertical bars on the left most top side 
     of the screen'
3. Scroll down to 'Lockscreen'
4. Under 'Lockscreen' enable 'Show song cover' feature
5. After the above step nothing happens after locking 
    the screen

### Expected behavior

After enabling 'Show song cover' feature whose functionality is to 'Use the playing song cover as the lockscreen wallpaper' as it is mentioned under the feature which should temporarily change the wallpaper of the lockscreen with each playing song and change according and stops until the app is closed or if no songs are played by the user's side

### Actual behavior

But after enabling the 'Show song cover' feature functionality is to 'Use the playing song cover as the lockscreen wallpaper' as it is mentioned under the feature, it does nothing just as if "it is a switch without any electricity connection given to it present in circuit, just present there taking up space without functioning"
-----In conclusion it does nothing, just flips from left to right that is it-----

### ViMusic version

0.5.4

### Android version

Android 12

### Additional information

Please repair this feature I love this feature a lot and as of now no other apps giving this feature according to me that is. I have last seen this feature on my Lenovo K8 Note Phone using Wynk Music App after that nowhere to be found.
-----I kindly request you all to please bring this feature and I would to use it------
Thank you for hearing my query
Yours Sincerely
     Suchait05
